apps/backend/src/lambdas/batch-reconcile.ts

```diff
@@ -1,16 +1,21 @@
 import { NestFactory } from '@nestjs/core';
 import { Context } from 'aws-lambda';
-import { eachDayOfInterval, format, subBusinessDays } from 'date-fns';
+import { eachDayOfInterval, format } from 'date-fns';
 import { generateLocalSNSMessage } from './helpers';
-import { HandlerEvent } from './interface';
+import { BatchHandlerEvent, ReconciliationEventMessage } from './interface';
 import { handler as reconcile } from './reconcile';
 import { AppModule } from '../app.module';
 import { AppLogger } from '../logger/logger.service';
 
 import { SnsManagerService } from '../sns-manager/sns-manager.service';
-
-/* eslint-disable @typescript-eslint/no-explicit-any */
-export const handler = async (event: HandlerEvent, _context?: Context) => {
+/**
+ * Enables batch reconciliation by iterating over the dates for a given period.
+ * Use batch.json to configure locally.
+ * Not used in production.
```

Comment:
Possible tech debt to log - if we don't intend this to be used in prod I think we might want to add a check and failure state here for the prod env so it can't fire by accident

Reply:
Hmm, good call. I mean, it _could_ be used in Prod, but it would have to be done so with intention. The only way to trigger this one (aside from a code-change) is to log into the prod aws lambda and "test" the lambda manually. I believe access to this env is limited (and whoever triggers it would also need to include correct inputs in order to run.) We may use this on tools/test/dev to test out new heuristics layers or code changes to reconciliation applied to the entire db, or a large chunk of it.

Reply:
I guess it's possible we would need this available on prod to recover from multiple days of missing files... maybe let's leave this for now until we have more clarity on how that case should be handled (there's already a ticket).